guide/creating-your-bot/creating-commands.md

```diff
@@ -81,96 +95,177 @@ const rest = new REST({ version: '9' }).setToken(token);
+:::
 
-You should first check if an interation is a command via <DocsLink path="class/Interaction?scrollTo=isCommand" type="method">`.isCommand()`</DocsLink>, and then check the <DocsLink path="class/CommandInteraction?scrollTo=commandName">`.commandName`</DocsLink> property to know which command it is. You can respond to interactions with <DocsLink path="class/CommandInteraction?scrollTo=reply">`.reply()`</DocsLink>.
+## Replying to Slash Commands
 
-```js {5-17}
+Once you've registered your Slash Commands, users can now use your Slash Commands! Our next step is to make our bot reply to those Slash Commands. Let's go back to our `index.js` file, and add these lines:
+
+:::: code-group
+::: code-group-item index.js
+```js {13-15}
+// Require the necessary discord.js classes
+const { Client, Intents } = require('discord.js');
+const { token } = require('./config.json');
+
+// Create a new client instance
+const client = new Client({ intents: [Intents.FLAGS.GUILDS] });
+
+// When the client is ready, run this code (only once)
 client.once('ready', () => {
 	console.log('Ready!');
 });
 
 client.on('interactionCreate', async interaction => {
-	if (!interaction.isCommand()) return;
-
-	const { commandName } = interaction;
-
-	if (commandName === 'ping') {
-		await interaction.reply('Pong!');
-	} else if (commandName === 'server') {
-		await interaction.reply('Server info.');
-	} else if (commandName === 'user') {
-		await interaction.reply('User info.');
-	}
+	console.log(interaction)
 });
 
+// Login to Discord with your client's token
 client.login(token);
 ```
+:::
+::::
+
+A Slash Command is just one type of [interaction](https://discord.com/developers/docs/interactions/application-commands#Slash-commands). Therefore, we can make our bot listen to the [`interactionCreate`](https://discord.js.org/#/docs/main/stable/class/Client?scrollTo=e-interactionCreate) event to receive incoming interactions. So far, we only have a `console.log(interaction)` in the callback. Go ahead and start the bot with:
+```sh:no-line-numbers
+node index.js
+```
+and then use the `/ping` command in your guild. You should see something logged in your terminal that is similar to this:
+```:no-line-numbers
+CommandInteraction {
+  type: 'APPLICATION_COMMAND',
+  id: '123',
+  applicationId: '123',
+  channelId: '123',
+  guildId: '123',
+  user: User { ... },
+  member: GuildMember { ... },
+  version: 1,
+  commandId: '123',
+  commandName: 'ping',
+  deferred: false,
+  options: CommandInteractionOptionResolver {
+    _group: null,
+    _subcommand: null,
+    _hoistedOptions: []
+  },
+  replied: false,
+  ephemeral: null,
+  webhook: InteractionWebhook { id: '123' }
+}
+```
+
+If you see something like that getting logged in your terminal, then that means your bot is receiving the interaction event correctly! However, our bot hasn't replied to the interaction yet, which is why you will see something like this in your Discord:
 
-### Server info command
```

Comment:
Why was this header removed?

Reply:
"server info command" and "user info command" header were removed because the `.reply()` of all three commands were implemented in the same code block. The purpose of this section is to show how to reply to a slash command, therefore it seemed appropriate to do so.